ui/dashboards/src/context/DashboardProvider/layout-slice.tsx

```diff
@@ -42,16 +36,19 @@ export interface LayoutSlice {
   /**
    * Updates an existing panel group to, for example, change its display properties.
    */
-  updatePanelGroup: (layout: LayoutDefinition, groupIndex?: number) => void;
+  updatePanelGroup: (layout: Omit<GroupDefinition, 'id'>, groupIndex?: number) => void;
 
   /**
    * Rearrange the order of panel groups by swapping the positions
    */
   swapPanelGroups: (xIndex: number, yIndex: number) => void;
 }
 
-export interface PanelGroupDefinition extends LayoutDefinition {
+export interface GroupDefinition {
```

Comment:
Any value in calling this something more specific like `PanelGroup` or `PanelGroupDefinition`? "Group" feels kinda vague to me, but could be fine.